frontend/Interfaces/lib/CMakeLists.txt

```diff
@@ -1,35 +1,37 @@
 if (${BUDDY_DIP_OPT_STRIP_MINING})
-  set(SPLITING_SIZE ${BUDDY_DIP_OPT_STRIP_MINING})
-elseif(HAVE_AVX512)
-  set(SPLITING_SIZE 256)
-elseif(HAVE_AVX2)
-  set(SPLITING_SIZE 128)
-elseif(HAVE_SSE)
-  set(SPLITING_SIZE 64)
-elseif(HAVE_NEON)
-  set(SPLITING_SIZE 64)
-endif()
+    set(SPLITING_SIZE ${BUDDY_DIP_OPT_STRIP_MINING})
+elseif (HAVE_AVX512)
+    set(SPLITING_SIZE 256)
+elseif (HAVE_AVX2)
+    set(SPLITING_SIZE 16)
```

Comment:
Did you compare performance between splitting size 16 and 128?

Reply:
We may encounter such a problem that optimization algorithms for different ops seem to need different configurations.

Reply:
I have compared performance on my own machine(with avx2) using stride = 2^k(k in [1, 8]). The best performance appears when stride=16 and 32. As Hongbin said, the best parameter differs on different machine, so I can only promise that this parameter is the best on _my_ machine.

Reply:
> optimization algorithms for different ops seem to need different configurations.

Is this behaviour dependent on the algorithm or the problem size itself (Image dimensions in this case) ?

Reply:
No. I have discussed with Hongbin on this issue and concluded that this parameter is to fully utilize the cpu's simd registers.